numaprom/udsink/train_rollout.py

```diff
@@ -45,15 +45,17 @@ def clean_data(df: pd.DataFrame, limit=12) -> pd.DataFrame:
         .drop("_merge", axis=1)
     )
     df.set_index("timestamp", inplace=True)
-    df.drop("hash_id", axis=1, inplace=True)
+    df.drop(hash_col, axis=1, inplace=True)
     df = df.sort_values(by=["timestamp"], ascending=True)
     if len(df) < (1.5 * 60 * 12):
         LOGGER.exception("Not enough training points to initiate training")
         return pd.DataFrame()
     return df
 
 
-def _fetch_data(metric_name: str, model_config: dict, labels: dict) -> pd.DataFrame:
+def _fetch_data(
```

Comment:
we could make this a common method, and use the same in both train and train_rollout

Reply:
done